make/platform/Makefile.cray-x-series

```diff
@@ -19,3 +19,9 @@ ifeq ($(CHPL_COMM), gasnet)
   GEN_CFLAGS += -I$(GASNET_INSTALL_DIR)/patched_headers
   GEN_LFLAGS += -L/usr/lib64/nptl
 endif
+
+# This ensures that the Chapel binary can find and link against the latest
+# gcc-lib versions. This addition was motivated by an error where chapel
+# could not find a version of GLIBCXX when compiling with intel/cray compilers
+# Note: This is assuming gcc-libs maintain forward compatibility
```

Comment:
You might also note that we got this from the CCE build/integration staff. If it breaks in the future due to for example someone changing the path we're using, we'll have a lead on who to start a conversation with.